hibike/lib/hibike/hibike_device.cpp

```diff
@@ -99,8 +98,8 @@ void hibike_loop() {
     send_data_update(params);
   }
 
-  if ((currTime - respHeartBeat)>disableLatency){
-    //disable(); //on sensor specific device'
+  if ((currTime - resp_heartbeat) > disable_latency) {
+    // disable(); //on sensor specific device'
```

Comment:
Now that disable() is implimented and merged to master, actually call disable()

This code is contained in the hibike library, E.G. a Smart Device developer should not be changing it.  If the smart Device developer doesn't want to have any disable functionality, they can instead leave disable() blank,